### PR TITLE
add bybit and bitmex

### PR DESCRIPTION
**Pull request by Define101**
add bybit and bitmex

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/bitmex exports TVL: 
        
 
 
    bitcoin                   237.04 M
    ethereum                  245.41 M
    
    total                    482.46 M

**Comment by llamatester** [comment]
Error while running adapter at projects/bybit: 
        
 
  
    
    
    TypeError: i.toLowerCase is not a function
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/utils.js:167:58
        at Array.forEach (<anonymous>)
        at getUniqueAddresses (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/utils.js:167:13)
        at sumTokens2 (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/unwrapLPs.js:977:14)
        at sumTokens (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/sumTokens.js:52:12)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/sumTokens.js:45:45
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:48:31)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:143:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:131:29

File: projects/bybit/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    owners: [
+        '31GrwDkr33gT6LuumniYjKEGjTLhsL5kmqC',
+        'bc1q2qqqt87kh33s0er58akh7v9cwjgd83z5smh9rp',
+        'bc1q9w4g79ndel72lygvwtqzem67z6uqv4yncvqjz3yn8my9swnwflxsutg4cx',
+        'bc1qjysjfd9t9aspttpjqzv68k0ydpe7pvyd5vlyn37868473lell5tqkz456m'
+           ]
+  },
+  binance: {
+    owners: [
+        ['0xee5B5B923fFcE93A870B3104b7CA09c3db80047A',]
```
Comment (g1nt0ki) [inline projects/bybit/index.js:23]: remove the `[]`

File: projects/bybit/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  bitcoin: {
+    owners: [
+        '31GrwDkr33gT6LuumniYjKEGjTLhsL5kmqC',
+        'bc1q2qqqt87kh33s0er58akh7v9cwjgd83z5smh9rp',
+        'bc1q9w4g79ndel72lygvwtqzem67z6uqv4yncvqjz3yn8my9swnwflxsutg4cx',
+        'bc1qjysjfd9t9aspttpjqzv68k0ydpe7pvyd5vlyn37868473lell5tqkz456m'
+           ]
+  },
+  binance: {
```
Comment (g1nt0ki) [inline projects/bybit/index.js:21]: should be `bsc`

File: projects/bybit/index.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    owners: [
+       '0x1Db92e2EeBC8E0c075a02BeA49a2935BcD2dFCF4', 
+       '0xA7A93fd0a276fc1C0197a5B5623eD117786eeD06',
+       '0xe1ab8c08294F8ee707D4eFa458eaB8BbEeB09215',
+       '0xee5B5B923fFcE93A870B3104b7CA09c3db80047A',
+       '0xf89d7b9c864f589bbF53a82105107622B35EaA40', // multiple chains
+    ],
+  },
+  bitcoin: {
+    owners: [
```
Comment (g1nt0ki) [inline projects/bybit/index.js:14]: one of these addresses has an issue, can you check on bitcoin explorer?

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The adapter at projects/bitmex exports TVL: 
        
 
 
    ethereum                  246.24 M
    bitcoin                   236.75 M
    
    total                    482.99 M

**Comment by llamatester** [comment]
The adapter at projects/bybit exports TVL: 
        
 
 
    bitcoin                   357.19 M
    tron                      129.56 M
    polygon                   10.70 M
    ethereum                  1.02 B
    bsc                       154.36 M
    
    total                    1.67 B

**Comment by llamatester** [comment]
The adapter at projects/bitmex exports TVL: 
        
 
 
    bitcoin                   236.78 M
    ethereum                  246.07 M
    
    total                    482.85 M

**Comment by llamatester** [comment]
The adapter at projects/bybit exports TVL: 
        
 
 
    bitcoin                   357.23 M
    tron                      129.45 M
    polygon                   10.69 M
    arbitrum                  53.83 M
    bsc                       154.37 M
    ethereum                  1.02 B
    
    total                    1.72 B

**Pull request merged**